### PR TITLE
bugfix/carrousel-landing

### PR DESCRIPTION
**Pull request by Sonny97**
change position to info description in carrousel and fix button-lockdown's size

Button-lockdown:
image

info carrousel center: 

image

### CODE REVIEW

**Comment by jusk02** [comment]
Please dont just copy and paste the css from zeplin, it is not optimized to work inside the site.

remove all the rules that set `normal` as these dont do anything.

and replace the runes like `color: var(--black);` with the correct color or remove them if they are not needed.

image

**Review by Sonny97 — COMMENTED** [review]
change the class and I make the necessary properties 🚀

File: css/style.css [hunk 1]
```diff
@@ -431,4 +431,29 @@ video {
...
+  .info {
+    margin-bottom: 18px;
+    width: 100%;
+    left: 0;
+    right: 0;
+    margin: auto;
+    text-align: center;
+  }
+  
+  .info-title {
```
Comment (camigomezdev) [inline css/style.css]: BEM!!!!!!!!!

File: css/style.css [hunk 2]
```diff
@@ -431,4 +431,29 @@ video {
...
+    width: 227px;
+    height: 24px;
+    font-size: 20px;
+    left: 0;
+    right: 0;
+    margin: auto;
+  }
+  
+  .info-description {
```
Comment (hanna446) [inline css/style.css]: I think this class should be called info__description

File: css/style.css (hunk 1 above)
Comment (Sonny97) [reply css/style.css]: changed to info__title

File: css/style.css (hunk 2 above)
Comment (Sonny97) [reply css/style.css]: changed, thanks to opinion

**Review by josemariogutierrez — APPROVED** [review]

**Review by jusk02 — APPROVED** [review]

**Pull request merged**